pkg/operator/util/ca.go

```diff
@@ -0,0 +1,78 @@
+package util
+
+import (
+	crand "crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"time"
+
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+const SigningCertificateLifetimeInDays = 365 // 1 year
+
+type SigningCA struct {
+	Config             *crypto.TLSCertificateConfig
+	Bundle             []*x509.Certificate
+	IntermediateCACert *x509.Certificate
+}
+
+// GetPEMBytes returns PEM-encodings of the CA cert, key, bundle and intermediate CA cert.
+func (ca *SigningCA) GetPEMBytes() ([]byte, []byte, []byte, []byte, error) {
```

Comment:
Inline this within `RotateSigningCA()`, the struct above is single-purpose, you're only carrying the bits over.

Reply:
Externalizing conversion to this method simplifies unit testing since conversion to PEM is unnecessary for the purposes of cert verification. Or would you like to see the exported method `RotateSigningCA` return PEM and an internal method `rotateSigningCA` that returns `SigningCA`?